demoapp/templates/project2.py

```diff
@@ -12,6 +12,7 @@ def error():
 
 @app.route('/')
 def index():
+    'rendering templates'

```

Comment:
Have some meaning full doc strings that explains functionality written in method

Reply:
Kk I will chek it sir